libraries/GCS_MAVLink/GCS.h

```diff
@@ -886,6 +888,28 @@ class GCS
     virtual MAV_TYPE frame_type() const = 0;
     virtual const char* frame_string() const { return nullptr; }
 
+    struct statustext_t {
+        uint8_t                 bitmask;
+        mavlink_statustext_t    msg;
+        uint16_t                entry_created_ms;
+    };
```

Comment:
```suggestion
    struct statustext_t {
        mavlink_statustext_t    msg;
        uint16_t                entry_created_ms;
        uint8_t                 bitmask;
    };
```

Shrinks the size of the struct, which in turn shrinks the size of the queue

Reply:
Fixed.